packages/debian/entrypoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+rpm2deb() {
+	set -x
+	chown build:build /home/build
```

Comment:
Why chowning the home directory, the ownership should already be right, no ?

Reply:
Indeed, It was more a kind of "security" to be sur. I delete it.